File: starters/python/mario-less/mario.py
# TODO: 实现 mario.py


def get_height():
    """获取用户输入的高度（1-8）"""
    while True:
        try:
            height = int(input("Height: "))
            if 1 <= height <= 8:
                return height
        except ValueError:
            pass


def main():
    height = get_height()

    # 打印半个金字塔
    pass


if __name__ == "__main__":
    main()

File: starters/java/mario-less/Mario.java
// TODO: 实现 Mario.java
import java.util.Scanner;

public class Mario {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // 获取用户输入的高度（1-8）
        int height;
        do {
            System.out.print("Height: ");
            height = scanner.nextInt();
        } while (height < 1 || height > 8);

        // 打印半个金字塔

        scanner.close();
    }
}
